### PR TITLE
add AMP playground

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: playground/serve.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		return
+	}
+	param := r.FormValue("url")
+	u, err := url.Parse(param)
+	if param == "" || err != nil || (u.Scheme != "http" && u.Scheme != "https") {
+		http.Error(w, "Bad request.", http.StatusBadRequest)
+		return
+	}
+	// only allow URLs from ampbyexample
+	if u.Host != "ampbyexample.com" &&
```
Comment (kul3r4) [inline playground/serve.go]: what about putting those 3 strings in an array of allowed URLs instead of enumerating those?

File: playground/serve.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	ctx := appengine.NewContext(r)
+	client := urlfetch.Client(ctx)
+	req, err := http.NewRequest("GET", u.String(), nil)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Bad gateway (%v)", err.Error()),
+			http.StatusBadGateway)
+		return
+	}
+	req.Header.Add("User-Agent",
+		"Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MTC19V) "+
```
Comment (kul3r4) [inline playground/serve.go]: why are these needed?

Comment (sebastianbenz) [inline playground/serve.go]: In case pages have a redirect for desktops.

File: playground/serve.go (hunk 1 above)
Comment (sebastianbenz) [inline playground/serve.go]: good idea. done.

**Comment by sebastianbenz** [comment]
PTAL

**Comment by kul3r4** [comment]
LGTM
